Wireframe/style.css

```diff
@@ -1,89 +1,68 @@
-/* Here are some starter styles
-You can edit these or replace them entirely 
-It's showing you a common way to organise CSS 
-And includes solutions to common problems 
-As well as useful links to learn more */
+/* General body styles */
+body {
+    font-family: Arial, sans-serif;
+    line-height: 1.6;
+    margin: 0;
+    padding: 0;
+    background-color: #ffffff;  /* white background */
```

Comment:
What value do you think these comments are adding? What would be worse if you removed them?

Please answer for _each_ comment - each thing we add to our code should have purpose, so presumably you had a reason for each one you added.